### PR TITLE
Add fingerprint fields for user id and event id.

### PR DESCRIPTION
**Pull request by tcsnfkx**
These fingerprints are only used by VID Labeler.

---
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/virtual-people-common/10)

### CODE REVIEW

**Review by lenjoy — APPROVED** [review]

**Review by efoxepstein — APPROVED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/virtual-people-common/10)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @EvgSkv, @kunyuchen, @mpatel24, @my2457, @oliver-amzn, and @wliue)

File: src/main/proto/wfa/virtual_people/common/event.proto [hunk 1]
```diff
@@ -30,6 +30,9 @@ message EventId {
   // The publisher specific event id. Unique within all logs of the
   // publisher.
   optional string id = 2;
+
+  // The fingerprint of id. Only used in Labeler.
```
Comment (wliue) [inline src/main/proto/wfa/virtual_people/common/event.proto:34]: Maybe "Only set and used in Labeler." to make it clear this field is not set in input events.

File: src/main/proto/wfa/virtual_people/common/event.proto [hunk 2]
```diff
@@ -52,6 +55,9 @@ message UserInfo {
   // Creation or initialization time of the user id, in microseconds
   // since epoch.
   int64 creation_time_usec = 5;
+
+  // The fingerprint of user_id. Only used in Labeler.
```
Comment (wliue) [inline src/main/proto/wfa/virtual_people/common/event.proto:59]: Maybe "Only set and used in Labeler." to make it clear this field is not set in input events.

**Review by wliue — APPROVED** [review]

File: src/main/proto/wfa/virtual_people/common/event.proto (hunk 1 above)
Comment (tcsnfkx) [reply src/main/proto/wfa/virtual_people/common/event.proto]: Done.

File: src/main/proto/wfa/virtual_people/common/event.proto (hunk 2 above)
Comment (tcsnfkx) [reply src/main/proto/wfa/virtual_people/common/event.proto]: Done.

**Pull request merged**